modules/http/src/main/java/org/mule/module/http/internal/listener/DefaultHttpListenerConfig.java

```diff
@@ -56,6 +56,7 @@
     private String basePath;
     private Boolean parseRequest;
     private MuleContext muleContext;
+    @Inject
     private HttpListenerConnectionManager connectionManager;
```

Comment:
A next step (if this PR is finally merged) is to replace all the usages of muleContext.getRegistry.lookup() with injection if possible